components/automate-ui/src/app/components/sidebar-entry/sidebar-entry.component.ts

```diff
@@ -13,6 +13,8 @@ export class SidebarEntryComponent {
   @Input() iconRotation: number;
   // (optional) if true, opens the link in a new page
   @Input() openInNewPage: boolean;
+  // (optional) shows custom icon
```

Comment:
👍 Good, but please add a similar line above `icon` on L9, something like:
```
// (optional) if present displays the standard icon
 @Input() icon: string;
```

Also, suggest putting the two declarations next to each other here (`icon` and `customIcon`) and please add a further comment that distinguishes the two. That is, when do I use `icon` and when do I use `customIcon` ? Like the casual reader, I actually do not know the answer as I write this. I would have to dig into the code to understand that--but let's help the reader so they do not have to. 😁 
(And a further question, coming from not yet digging into the code: do we really need both of them?)

Reply:
@msorens yes we need both of them. `icon` is used to show material icon and `customIcon` is used when there is no respective material icon is available. I took reference from Waiver (#3089).